src/pages/workspaces/workspace/workflows/LaunchAnalysisModal.js

```diff
@@ -91,9 +91,9 @@ const LaunchAnalysisModal = ({
         memoryRetryMultiplier: retryWithMoreMemory ? retryMemoryFactor : undefined,
         userComment: _.trim(userComment),
         ignoreEmptyOutputs,
-        monitoringScript,
-        monitoringImage,
-        monitoringImageScript,
+        monitoringScript: monitoringScript || null,
+        monitoringImage: monitoringImage || null,
+        monitoringImageScript: monitoringImageScript || null,
```

Comment:
Is there a reason we would set them to null here, rather than `WorkflowView.js`, where we check the stored state and fall back to an empty string?